### DISCUSSION TITLE
Why cant I make geist button type="submit"?

### DISCUSSION THREAD

**Discussion by BekaArabidze** [zeit-ui/react#449] [Q&A] [answered]
Hello. I want to use type="submit" in my button component but geist does not let me do it. Also, I want to make my input component type=" number", but I ran a similar problem did I miss something or I cant make the geist button type="submit"? Thanks for helping

**Comment by BekaArabidze** [comment]
ah. Forget about this. I found it :ddddd

**Comment by ofekashery** [comment, marked as answer]
As [documented](https://react.geist-ui.dev/en-us/components/button#apis), you should use the ``htmlType`` attribute to set the ``Button`` HTML type. For example:
```jsx
<Button type="success" htmlType="submit">Success</Button>
```

<br />

For ``Input``, use the regular ``type`` attribute:
```jsx
<Input placeholder="Number Input" type="number" />
```

**Reply by BekaArabidze** [reply]
Thank you. I found it. <3
